images/ose-installer-altinfra.yml

```
@@ -0,0 +1,26 @@
+content:
+  source:
+    dockerfile: images/installer/Dockerfile.ci
```

https://github.com/openshift/installer/pull/7711 would change this to
```suggestion
    dockerfile: images/installer-altinfra/Dockerfile.ci
```

---

The PR above has merged, so this should be good to  be updated.